controlplane/src/core/env.schema.ts

```diff
@@ -105,8 +105,26 @@ export const envVariables = z
     /**
      * Email
      */
+    SMTP_ENABLED: z
+      .string()
+      .transform((val) => val === 'true')
+      .default('false'),
     SMTP_USERNAME: z.string().optional(),
     SMTP_PASSWORD: z.string().optional(),
+    SMTP_HOST: z.string().optional(),
+    SMTP_PORT: z
+      .string()
+      .optional()
+      .default('') // here to avoid possible type of number | "" | undefined type
```

Comment:
@StarpTech you might have a better solution for this.

If I leave away the default, tsc interferes `number | "" | undefined` as possible type as I would have to do something like this:

```
.transform((val) => val && Number.parseInt(val)),     
```

As val has then no explicit type. I could also handle it throughout the interfaces, but it feels bad, as all other types are usually `undefined | string` or `undefined | number` or such.

Reply:
Do we need a default?

Reply:
```ts
      SMTP_PORT: z
          .coerce.number().optional()
```

Reply:
Thank you, this will now avoid the default.